README.md

```diff
@@ -318,6 +318,9 @@ instructions below may not work properly on Windows.
 
 * If you want to make an executable, simply run `./scripts/build`. The binary will appear in the `./out` folder.
 
+## Security
```

Comment:
I would rather say:

"
The CenturyLink Cloud Go CLI leverages our [public API](//www.ctl.io/api-docs/v2/) that serves all requests over HTTPS. Therefore credentials are encrypted when being transfered. Credentials stored on the local machine in the [config.yml](#Set-up-the-configuration-file) are not encrypted during installation and you are encouraged to use industry standard encryption tools in order to provide additional protection for them.
"

If it does not sound reasonable to you, we could use your version - it's ok.